### ISSUE TITLE
Android build fails because CocoaPods was not found??

### ISSUE THREAD

**Issue by Bubelbub** [OneSignal/OneSignal-Cordova-SDK#225]
Hey guys,

do someone know how to solve this problem?

    $ ionic cordova build android --prod --release --no-interactive --confirm

This leads to this cordova command.

    $ cordova prepare

Which ends with

	Discovered plugin "onesignal-cordova-plugin" in config.xml. Adding it to the project
	Installing "onesignal-cordova-plugin" for android
	Subproject Path: CordovaLib
	Installing "onesignal-cordova-plugin" for ios
	apple-xcode version check failed ("/bitrise/src/platforms/ios/cordova/apple_xcode_version"), continuing anyways.
	apple-ios version check failed ("/bitrise/src/platforms/ios/cordova/apple_ios_version"), continuing anyways.
	Failed to install 'onesignal-cordova-plugin': undefined
	Failed to restore plugin "onesignal-cordova-plugin" from config.xml. You might need to try adding it again. Error: CocoaPods was not found. Please install version 1.0.1 or greater from https://cocoapods.org/

But I not want the iOS platform there because it's no Mac OS and there is no XCode and nothing.

How can I avoid that `cordova prepare` loads the `onesignal-cordova-plugin` from iOS?

[config.xml](https://github.com/OneSignal/OneSignal-Cordova-SDK/files/1229784/config.xml.txt)
[ionic.config.json](https://github.com/OneSignal/OneSignal-Cordova-SDK/files/1229786/ionic.config.json.txt)
[package.json](https://github.com/OneSignal/OneSignal-Cordova-SDK/files/1229782/package.json.txt)
[tsconfig.json](https://github.com/OneSignal/OneSignal-Cordova-SDK/files/1229785/tsconfig.json.txt)

☺️ **Thank's in Advance for any hints!** ☺️

**Comment by jkasten2** [comment]
@Bubelbub It looks like the iOS platform was added to your project. You can remove it by running the following.
```Shell
cordova platform remove ios
```

**Comment by Bubelbub** [comment]
Ah ok @jkasten2 but the main problem was that I never added this platform because I'm developing on Windows and there is no possibility to add iOS platform ^^

But the hint was good. After I did what you said I get a lot of errors that the directory can't be removed (because its not exist) AND a success message.

    Removing platform ios from config.xml file...
    Removing ios from cordova.platforms array in package.json

And then I checked my git status and saw.

![2017-08-17 12_25_34](https://user-images.githubusercontent.com/319352/29408201-9b3241ec-8347-11e7-8bde-bc1ebeb8c633.png)

That the only problem is the `package.json` which contains at the end this `platforms` key which contains `android` and `ios`. After removing this it works well 👍
